src/javascript/node/sessionless.js

```diff
@@ -31,10 +55,16 @@ const getKeys = async () => {
   }
 };
 
-const sign = (message) => {
-  const { privateKey } = getKeys();
-  const messageHash = keccak256(utf8ToBytes(message));
-  return secp256k1.sign(messageHash, privateKey);
+const sign = async (message) => {
```

Comment:
seems fine overall
I could only think of input validation here since `message` is being used without making sure it's a valid string which can cause some errors to be emitted from other functions (like `.slice(..)`) when the real issue is the input being invalid to begin with

But this is not a big deal tbh

Reply:
I did think about that, and Ethereum does do message validation, but I felt that if we put validation here we'd have to go 100% or it'd be too confusing, and since that would require a lot of validation across all the languages I left it out.